### ISSUE TITLE
Add fan control utility 

### ISSUE THREAD

**Issue by anujdeshpande** [balena-os/balena-raspberrypi#602]
RPi4 comes with a case + fan, and also a utility to configure this (temperature threshold, etc). Something like this in balenaOS would be nice so that one can use the official hardware from Pi foundation similar to raspbian.

**Comment by uxsoft** [comment]
Agree, same issue here, the fan is running 100% even if the Pi is idle which is annoying

**Comment by Jeff254** [comment]
Any Traction with this?

**Comment by alexgg** [comment]
hey @Jeff254 unfortunately feature requests in a github issue does not get to our product team. Could I ask you to open a proper feature request in https://feature-requests.balena.io instead? Thanks for your understanding.
